### DISCUSSION TITLE
Preinstall Dart on new CodeSpaces.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#10109] [Codespaces] [answered]
Hello,
Please include Dart in all new codespaces, as it is becoming more popular and more people are using it. This would be a good option that will save users hundreds of hours of time and eliminate the need to worry about downloading a language.

Simple Math:
If each user takes 5 minutes to install Dart, and there are 20,000 individuals who need to install it on various codespaces, we'll have spent over 100000 minutes simply installing it.

- prajwal

**Comment by shellerci** [comment, marked as answer]
Have you tried basing off of the  dart devcontainer definition? 

https://github.com/microsoft/vscode-dev-containers/tree/main/containers/dart
